### PR TITLE
Feature/playservices attempt2

### PR DESCRIPTION
**Pull request by andaag**
Another attempt at getting playservices - much cleaner api imo

### CODE REVIEW

File: sample/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,7 +14,8 @@
         android:allowBackup="false">
 
         <activity android:name="no.finntech.rxactivityresponse.sample.SerializedRxSampleActivity"
-                  android:label="Sample">
+                  android:label="Sample"
```
Comment (haakoneriksen) [inline sample/src/main/AndroidManifest.xml:17]: Navnet "Sample" gjør den vanskelig å finne i apps-lista når man skal skru av permissions for å teste

File: playservices/src/main/java/no/finntech/android/rx/LocationSettingOperator.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                        final Status status = result.getStatus();
+                        switch (status.getStatusCode()) {
+                            case LocationSettingsStatusCodes.SUCCESS:
+                                subscriber.onNext(client);
+                                break;
+                            case LocationSettingsStatusCodes.RESOLUTION_REQUIRED:
+                                try {
+                                    RxActivityResponseDelegate rxActivityResponseDelegate = RxActivityResponseDelegate.get(activity);
+                                    if (!rxActivityResponseDelegate.hasActiveResponse()) {
+                                        rxActivityResponseDelegate.setResponse(responseHandler);
```
Comment (haakoneriksen) [inline playservices/src/main/java/no/finntech/android/rx/LocationSettingOperator.java:61]: At this point we are inside an if-statement which is inside a try/catch-statement which is inside a switch statement which is inside a resultcallback which is inside a subscriber which is inside a subscriber, which is probably being called from a click listener. If this is made to be as simple as possible, it feels as if someone (either we or google) is doing something wrong.

Comment (andaag) [inline playservices/src/main/java/no/finntech/android/rx/LocationSettingOperator.java:61]: I'm not too  happy about that either. Tried to make it simpler by allowing rxActivityResponseDelegate.handleResponse(interface that's called if we dont already have an active response), but that didn't look too good either.

File: sample/src/main/AndroidManifest.xml (hunk 1 above)
Comment (andaag) [inline sample/src/main/AndroidManifest.xml:17]: Making a seperate PR with some cleanup for the rest of the code.

**Pull request merged**